modules/tinymce/src/plugins/image/test/ts/browser/core/ImageSelectionTest.ts

```diff
@@ -210,4 +210,80 @@ describe('browser.tinymce.plugins.image.core.ImageSelectionTest', () => {
       });
     }));
   });
-});
+  it('If <figure> is not valid child, it should split parent node', async () => {
```

Comment:
While it's not apparent in this test file, as all the tests are really old, about 1yr ago we started including the JIRA number in the test title. This helps QA to see which tests link to specific JIRAs and also makes it easy to trace back if we need to determine later why a test was added.

Also as a side note/nitpick, please make sure to include a blank line between tests as it makes it more readable since it breaks up the content.

```suggestion

  it('TINY-6592: If <figure> is not valid child, it should split the parent node', async () => {
```